db/migrate/20240422080526_add_status_to_users.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
why are you using it here?

Reply:
@kth-wtag apu, I get style warning if I do not use it.

Reply:
> @kth-wtag apu, I get style warning if I do not use it.

It shouldn't be. which version of rubocop are you using?

Reply:
`1.62.1` @kth-wtag apu.

Reply:
> > @kth-wtag apu, I get style warning if I do not use it.
> 
> It shouldn't be. which version of rubocop are you using?

`1.62.1` @kth-wtag apu.

Reply:
> > > @kth-wtag apu, I get style warning if I do not use it.
> > 
> > 
> > It shouldn't be. which version of rubocop are you using?
> 
> `1.62.1` @kth-wtag apu.

```
Style/FrozenStringLiteralComment:
  Description: Add the frozen_string_literal comment to the top of files to help transition from Ruby 2.3.0 to Ruby 3.0.
  Enabled: false
```
add this .rubocop.config file

Reply:
@kth-wtag Apu, added.